### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by wadaudzai**
This part of the project was created to add some accessibility changes to the project.
The following list is created to point out which accessibility evaluations were done and what is their result.
1: Page Titles: Change the page title to the appropriate name.
2: Image text alternatives: No Issues Found.
3: Text Headings: No issues found.
4: Color Contrast: Change colors to make the required contrast: 4.5:1
5: Resize: Change certain widths and heights in order to fix the overlapping issue.
6: Interaction: No issues found.
7: Moving content: No issues found.
8: Multimedia: No issues found.
9: The Basic Structure of the page: No issues found.

### CODE REVIEW

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @wadaudzai @renercrows , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great title and short summary 
✔️ Github flow 

## Required Changes ♻️

- [ ] please it would be better if you wrap your navigation links with a `<a>` tag in order to be accessible with the keyboard so people with disability can access the the navigation links
image

- [ ] please make sure that only one `<h1>` in your portfolio so try to change others to `<h2....6>` in order to improve Best practices and SEO benchmarks
- [ ] - [ ] please it would be better if you wrap your `<input>` tags in the contact section with the `<lable>` tag and don't forget to use the `for` and `id` attributes that will help the search engine to recognize your inputs 


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -17,9 +17,9 @@
         <nav class="menu">
             <img src="./assets/icons/Union.png" alt="Menu Icon" class="header-image-mobile">
             <ul class="top-nav-desktop">
-                <li class="top-nav-items-desktop">Portfolio</li>
-                <li class="top-nav-items-desktop">About</li>
-                <li class="top-nav-items-desktop">Contact</li>
+                <a href="#" class="top-nav-items-desktop"><li>Portfolio</li></a>
+                <a href="#" class="top-nav-items-desktop"><li>About</li></a>
+                <a href="#" class="top-nav-items-desktop"><li>Contact</li></a>
```
Comment (youmari) [inline index.html]: - [ ] Good implement the changes asked on previous review but you should keep the `<li>` in the `<ul>` like that 
example : 
```HTML
 <ul>
        <li>
           <a href="#" >Portfolio</a>
        </li>
        <li>
          <a href="#" >About</a>
         </li>
         <li>
           <a href="#" >Contact</a>
         </li>
</ul>

```

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @wadaudzai , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights
✔️ Great title and short summary 
✔️ Good commit messages
✔️ Github flow 

## Required Changes ♻️

- [ ] [OPTIONAL]please try to add some padding to the button in order to match the Figma design 
image

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**